### PR TITLE
[EDU-2219] feat: Suite Static Cache template guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-2219] 

### Changes

- Adding guides on how to use the Suite Static Cache template
- 
### Additional links

n/a.

[EDU-2219]: https://aziontech.atlassian.net/browse/EDU-2219?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+<Badge variant="accent">
+Preview
+</Badge>
+
+The **Suite Static Cache** template includes configurations for optimizing the delivery and performance of static content, such as JavaScript, CSS, and other static files. These configurations use advanced cache key rules that are applied through **Rules Engine for Edge Application**.
+
+You must link your origin with the template to start using this set of cache rules for delivery. After the deployment, the script creates a new edge application that runs on Azion's Edge Computing Platform, with which you can manage and adjust the settings.
+
+This template:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx:19]:
```suggestion
This template:

```

File: src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+You must activate these modules separately via RTM. To do so:
+
+1. [Access RTM](https://manager.azion.com/).
+2. On the upper-left corner, select the three horizontal lines to open the **Products menu**.
+3. In the **BUILD** section, select **Edge Application**.
+4. You'll be redirected to the **Edge Application** page.
+- It lists all the edge applications you've created. 
+5. Find the edge application related to your template and select it. 
+6. In the **Main Settings** tab, find the **Edge Application Modules** section and activate the switch for the product you want to enable.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx:92]:
```suggestion
- You'll be redirected to the **Edge Application** page. It lists all the edge applications you've created. 
4. Find the edge application related to your template and select it. 
5. In the **Main Settings** tab, find the **Edge Application Modules** section and activate the switch for the product you want to enable.
```

File: src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+To manage and edit your edge application's settings, proceed as follows:
+
+1. [Access RTM](https://manager.azion.com/).
+2. On the upper-left corner, select the three horizontal lines to open the **Products menu**.
+3. In the **BUILD** section, select **Edge Application**.
+4. You'll be redirected to the **Edge Application** page.
+- It lists all the edge applications you've created. 
+5. Find the edge application related to the Hello World template and select it. 
+- The list is organized alphabetically. You can also use the **search bar** located in the upper-left corner of the list; currently, it filters only by **Application Name**.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx:112]:
```suggestion
- You'll be redirected to the **Edge Application** page. It lists all the edge applications you've created. 
4. Find the edge application related to the Hello World template and select it. 
- The list is organized alphabetically. You can also use the **search bar** located in the upper-left corner of the list; currently, it filters only by **Application Name**.
```

**Review by hannah-sk — APPROVED** [review]
algumas sugestões + 1 espaçamento

File: src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+permalink: /documentation/products/guides/suite-static-cache/
+---
+
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Preview
+</Badge>
+
+The **Suite Static Cache** template includes configurations for optimizing the delivery and performance of static content, such as JavaScript, CSS, and other static files. These configurations use advanced cache key rules that are applied through **Rules Engine for Edge Application**.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/marketplace/templates/static-template/static-template.mdx:15]:
```suggestion
The **Suite Static Cache** template includes configurations that optimize the delivery and performance of static content, such as JavaScript, CSS, and other static files. These configurations use advanced cache key rules that are applied through **Rules Engine for Edge Application**.
```

**Pull request merged**